### ISSUE TITLE
New REST URL format documentation / use errors

### ISSUE THREAD

**Issue by jimsiff** [nightscout/android-uploader#134]
Users who remove old uploader apps then manually install of the 0.1.12 uploader via APK rather than upgrade through the Play Store are having problems with the new REST URL format.

1. Users aren't aware of the syntax change.  The new format isn't called out in the release notes and wasn't mentioned in the uploader announcement.  It's also not documented in the existing Labs.  Greg or I will update the REST and Heroku labs, but the release notes and communication to endusers should mention the format change.

2. The auto-reformat logic works well for users updating via the Play Store, but it would be ideal if the same logic could be applied to manually entered URLs for this and maybe the next release until most users are beyond Brownie and Cookie Monster and accustomed to the new format.  Accepting but deprecating syntax can be a more user-friendly approach to command changes.

3.  It would be more user-friendly if the preferences logic could return the user to their previously entered text rather than a blank text box when there is a syntax error.  It would make it easier to find typos and possibly avoid re-entry of a long REST URL array.  I mentioned this to @trhodeos in the FB support page but I'm calling it out here for completeness.

**Comment by jasoncalabrese** [comment]
What do we need to do to really start using http://nightscout.github.io/pages/configure/?  When we messaged about it last night I think you were saying the labels are needed to be updated.

**Comment by jimsiff** [comment]
I agree the configure tool will make things a lot easier.  I'll open an issue with a few suggestions for the configure page.

**Comment by trhodeos** [comment]
Everything should work on the configure page. There may be some extra tabs that we need to remove.. but mongodb and the rest api tabs should work with the v0.1.12 release.
